site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,522 @@
+
+<Image
+  src="/images/patterns/navigation/examples/navigation-pattern-component-order.png"
+  alt="An illustration of the recommended order of navigation components. The order is Header, Breadcrumbs, Hero, Main content and Footer."
+  style="max-width:480px;"
+  viewable
+/>
+
+### 1. Header
+
+The navigation components are placed at the very top of the page and serves as the primary navigation area for your website or application. <strong>The Header component is a wrapper for the HDS header navigation components.</strong> This means you should put all the other header sub components inside this component.
+
+The header's consistent placement and appearance across all pages make it easily recognizable and accessible to users, allowing them to navigate the site quickly and efficiently.
+
+<PlaygroundPreview>
+  <Header>
+    <Header.UniversalBar primaryLinkText="Helsingin kaupunki" primaryLinkHref="#">
+      <Header.NavigationLink href="#" label="Link 1" />
+      <Header.NavigationLink href="#" label="Link 2" />
+      <Header.NavigationLink href="#" label="Link 3" />
+    </Header.UniversalBar>
+  </Header>
+</PlaygroundPreview>
+
+<ul style="margin-bottom:0;">
+  <li>
+    The inner components <strong>should always</strong> be in this order:
+    <ol>
+      <li>Header.UniversalBar</li>
+      <li>Header.ActionBar</li>
+      <li>Header.NavigationBar</li>
+    </ol>
+  </li>
+  <li>
+    The learn more about the <strong>Header</strong> component, please refer to
+    <InternalLink href="/components/header">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 2. Breadcrumbs
+
+Breadcrumbs are a secondary navigation component that helps users understand their current location within the site hierarchy and offers a simple way to navigate back to previous pages. Placed immediately below the header, breadcrumbs provide context and enhance usability by offering a clear path through the website or application.
+
+<PlaygroundPreview>
+  <Breadcrumb
+    ariaLabel="Breadcrumb"
+    list={[
+      { title: 'Front page', path: '/' },
+      { title: 'Health and social services', path: '/path' },
+      { title: 'Senior services', path: '/path/2ndLevelPath' },
+      { title: 'Informal care', path: '/path/2ndLevelPath/3rdLevelPath' },
+      { title: 'Care options', path: null },
+    ]}
+  ></Breadcrumb>
+</PlaygroundPreview>
+
+<ul style="margin-bottom:0;">
+  <li>
+    The learn more about the <strong>Breadcrumb</strong> component, please refer to
+    <InternalLink href="/components/breadcrumb">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 3. Hero
+
+The hero section is an optional but highly recommended component that usually appears below the breadcrumbs. It serves as a visual and content-focused introduction to the page, typically including a large image, heading, and a call-to-action button or message. The hero section helps grab the user's attention and sets the tone for the rest of the page.
```

Comment:
Hero should only be used on landing pages, not on all pages.

Reply:
Yup! Thanks!